### ISSUE TITLE
Adding support for BlackBerry Launcher

### ISSUE THREAD

**Issue by dalight13** [danimahardhika/candybar-library#89]
Is there any way I can modify LauncherHelper.java? If so how? Do I have to import library into my app or can I just add a LauncherHelper.java class into my own app?

I'm trying to add BlackBerry Launcher : https://play.google.com/store/apps/details?id=com.blackberry.blackberrylauncher

I looked into LauncherHelper.java and found out that it works perfectly with "applyManual", If I can't modify it is there any way you could add it please?

I tested my Icon Packs with the BlackBerry Launcher and they work perfectly

Thank you in advance

daLight

**Comment by dalight13** [comment]
QUOTE "You should open a new issue instead. Blackberry launcher doesn't  support direct or manual apply from the dashboard. The only way to apply  icon pack is go to settings and apply icon pack from there."

Thanks for your response, isn't that what the applyManual inside LauncherHelper.java is for? It does a popup saying to go to launcher settings? Because that is what I want to do.

**Comment by danimahardhika** [comment]
Yes, but blackberry launcher settings can't be accessed from the dashboard.

**Comment by danimahardhika** [comment]
Yes, evie settings can't be accessed.

I'll considering about this.

**Comment by dalight13** [comment]
Thank you very much!

Just so you know, in the BlackBerry Launcher, to apply the pack it is exactly "tap and hold homescreen - settings - iconpack - iconpackname"

So just applying the same popup as Evie would make it perfect. 

Thank you very much for your great work!

daLight
